src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/BindableTypeProvidersGenerationTask.cs

```diff
@@ -85,9 +84,8 @@ internal void Generate(GeneratorExecutionContext context)
 						_assemblyName = context.GetMSBuildPropertyValue("AssemblyName");
 
 						_defaultNamespace = context.GetMSBuildPropertyValue("RootNamespace");
-						_namedSymbolsLookup = context.Compilation.GetSymbolNameLookup();
 
-						_bindableAttributeSymbol = FindBindableAttributes();
+						_bindableAttributeSymbol = context.Compilation.GetTypeByMetadataName(XamlConstants.Types.BindableAttribute);
```

Comment:
We should also be searching for `System.ComponentModel.BindableAttribute`, as a compatibility measure as there are some non-winui viewmodels that use it.